### PR TITLE
Cloud package release

### PR DESCRIPTION
**Pull request by meker12**
## Purpose of this pull request

Added release notes for the following Cloud Suite packages:
- ece-tools v2002.0.23
- Cloud Patches v1.0.2
- Cloud Components v1.0.3

## Affected DevDocs pages

- https://devdocs.magento.com/cloud/release-notes/ece-release-notes.html
- https://devdocs.magento.com/cloud/release-notes/mcp-release-notes.html
- https://devdocs.magento.com/cloud/release-notes/mcc-release-notes.html

whatsnew
Added release notes for the following Magento Cloud Suite packages:  [ece-tools v2002.0.23](https://devdocs.magento.com/cloud/release-notes/ece-release-notes.html), 
[Magento Cloud Patches v1.0.2](https://devdocs.magento.com/cloud/release-notes/mcc-release-notes.html), and [Magento Cloud Components v1.0.3](https://devdocs.magento.com/cloud/release-notes/mcc-release-notes.html).

### CODE REVIEW

File: src/cloud/release-notes/ece-release-notes.md [hunk 1]
```diff
@@ -73,6 +72,11 @@ See [Upgrades and patches]({{ site.baseurl }}/cloud/project/project-upgrade-pare
 
 -  {:.fix}Fixed an issue that caused the build process to fail when the `config.php` file is empty.<!--MAGECLOUD-4127-->
 
+## 2002.0.23
+*Release date: February 24, 2020*<br/>
+
+-  {:.fix}Fixed a compatibility issue with `{{site.data.var.ct}}` 2002.0.x releases that prevented on demand static content generation from completing successfully in Magento production mode.
```
Comment (hguthrie) [inline src/cloud/release-notes/ece-release-notes.md]:
```suggestion
-  {:.fix}Fixed a compatibility issue with `{{site.data.var.ct}}` 2002.0.x releases that prevented on-demand static content generation from completing successfully in Magento production mode.
```

File: src/cloud/release-notes/ece-release-notes.md [hunk 2]
```diff
@@ -73,6 +72,11 @@ See [Upgrades and patches]({{ site.baseurl }}/cloud/project/project-upgrade-pare
 
 -  {:.fix}Fixed an issue that caused the build process to fail when the `config.php` file is empty.<!--MAGECLOUD-4127-->
 
+## 2002.0.23
+*Release date: February 24, 2020*<br/>
```
Comment (meker12) [inline src/cloud/release-notes/ece-release-notes.md]:
```suggestion
*Release date: February 27, 2020*<br/>
```

File: src/cloud/release-notes/mcc-release-notes.md [hunk 3]
```diff
@@ -11,17 +11,25 @@ The [Magento Cloud Components](https://github.com/magento/magento-cloud-componen
 
 The `{{site.data.var.mcc}}` package uses the following version sequence: `<major>.<minor>.<patch>`.
 
+## v1.0.3
+*Release date: February 24, 2020*<br/>
```
Comment (meker12) [inline src/cloud/release-notes/mcc-release-notes.md]:
```suggestion
*Release date: February 27, 2020*<br/>
```

**Comment by meker12** [comment]
running tests

**Pull request merged**